app/models/miq_alert.rb

```diff
@@ -26,6 +26,7 @@ class MiqAlert < ApplicationRecord
     ExtManagementSystem
     MiqServer
     ContainerNode
+    ContainerProject
```

Comment:
![screencapture-localhost-3000-miq_policy-explorer-1517736598928](https://user-images.githubusercontent.com/11256940/35776097-caac37c4-099e-11e8-901b-b6c1baae397f.png)

Also available in alerts